cime_config/tests.py

```diff
@@ -52,6 +52,7 @@
             "ERS.ELM_USRDAT.I1850CNPRDCTCBC.elm-snowveg_arctic",
             "ERS.ELM_USRDAT.I1850CNPRDCTCBC.elm-usrpft_default_I1850CNPRDCTCBC",
             "ERS.ELM_USRDAT.I1850CNPRDCTCBC.elm-usrpft_codetest_I1850CNPRDCTCBC",
+            "ERS.ELM_USRDAT.I1850CNPRDCTCBC.elm-polygonal_tundra"
```

Comment:
Since `1x1pt_icycape` is a supported `-res,` how about out changing this test to be `ERS.1x1pt_icycape.I1850CNPRDCTCBC.elm-polygonal_tundra`?

Reply:
Done